.travis.yml

```diff
@@ -15,8 +15,10 @@ before_script:
 #  - sudo jpm install com.codacy:codacy-coverage-reporter:assembly
 
 env:
-  - TEST_SUITE="travis_wait gradle check --stacktrace"   # unit tests
+  - TEST_SUITE="travis_wait gradle test --stacktrace"   # unit tests
   - TEST_SUITE="gradle fullAcceptanceTest --stacktrace"  # acceptance tests
+  - TEST_SUITE="gradle checkstyleMain --stacktrace"  # checkstyle main
```

Comment:
Maybe it is better to execute checkstyle first to not wait for fullAcceptanceTest if your checkstyle is inappropriate?